packages/@aws-cdk/aws-cloudwatch/README.md

```diff
@@ -41,6 +41,20 @@ const metric = new Metric({
 });
 ```
 
+You can also instantiate `Metric` objects to reference metrics
+[published by AWS services](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/aws-services-cloudwatch-metrics.html). For example:
+
+```ts
+const queue = new sqs.Queue(this, 'MySqsQueue');
+const metric = new Metric({
+  namespace: 'AWS/SQS',
```

Comment:
The examples in the README referred to custom metrics, so I wasn't sure how to work with existing AWS service metrics. My thought is that this example provides a simple way for others to discover how to do this.

Reply:
I'm not opposed to adding an additional example, but you are aware that for this particular example, SQS Queues already provide an accessor for this metric, right?

https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-sqs.Queue.html#metric-wbr-approximate-wbr-number-wbr-of-wbr-messages-wbr-visibleprops

So in this case, I would recommend writing:

```
queue.metricApproximateNumberOfMessagesVisible()
```

I'd rather use an example for which we don't have these accessors yet, and explicitly note in the accompanying text that you should only need to do this if and when we don't have them.

(And then also pretty please submit a PR adding the metric accessor ;) )

Reply:
Thanks for this guidance - I tried to tackle both concerns with this single PR, with a section on referencing metrics from other constructs. I placed this before the existing block with custom metrics, since I believe accessing metrics published by CDK constructs is the most common use-case for most.